integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/RangerContainer.java

```diff
@@ -38,7 +38,7 @@ public class RangerContainer extends BaseContainer {
   public static final String DEFAULT_IMAGE = System.getenv("GRAVITINO_CI_RANGER_DOCKER_IMAGE");
   public static final String HOST_NAME = "gravitino-ci-ranger";
   public static final int RANGER_SERVER_PORT = 6080;
-  public RangerClientExtension rangerClient;
+  public RangerClient rangerClient;
```

Comment:
I saw that this variable was originally used `RangerClient`, then you changed it to `RangerClientExtension`, why did you change it back to `RangerClient`? Did I miss something?

Reply:
Because `RangerClientExtension` in the `authorization-ranger` module.
I want the `integration-test-common` not to depend `authorization-ranger` module.